test/e2e/testsuite/testsuite.go

```diff
@@ -808,10 +810,12 @@ func EnsureCRCIsRunning() error {
 			return err
 		}
 
-		// set up and start the cluster with lots of memory
-		err = SetConfigPropertyToValueSucceedsOrFails("memory", "12000", "succeeds")
-		if err != nil {
-			return err
+		// set up and start the cluster with lots of memory, if specified
+		if CRCMemory != "" {
+			err = SetConfigPropertyToValueSucceedsOrFails("memory", CRCMemory, "succeeds")
+			if err != nil {
+				return err
+			}
```

Comment:
with this change looks like the cluster is going to always use default until specified with arg so how does this replicate setting the memory to `12000` or we want to run the cluster with default memory option?

Reply:
Got it, I think our default is only for play with openshift and have very limited workload, if someone wants to use pipeline then better to go with high mem/cpus.